examples/gno.land/p/demo/grc/grc721/basic_nft.gno

```diff
@@ -219,8 +224,12 @@ func (s *basicNFT) Burn(tid TokenID) error {
 	s.balances.Set(owner.String(), balance)
 	s.owners.Remove(string(tid))
 
-	event := TransferEvent{owner, zeroAddress, tid}
-	emit(&event)
+	std.Emit(
+		BurnEvent,
+		"from", string(owner),
+		"to", "",
```

Comment:
This can be omitted, the `to`

Reply:
65eb0b4aeb310b26ac6bb653e6cffe7e295be485